dapps/js-glue/src/codegen.rs

```diff
@@ -101,6 +101,7 @@ fn implement_webapp(cx: &ExtCtxt, builder: &aster::AstBuilder, item: &Item, push
 
 	let files_impl = quote_item!(cx,
 		impl $type_name {
+			#[warn(unused_mut)]
```

Comment:
ishould it be `#[allow(unused_mut)]`?

Reply:
Weirdly enough it did not produce any warnings when building like this.